app/models/compute_resources/foreman/model/ovirt.rb

```diff
@@ -30,6 +30,42 @@ def supports_update?
       true
     end
 
+    def supports_operating_systems?
+      client.respond_to?(:operating_systems) || client.send(:client).respond_to?(:operating_systems)
```

Comment:
please add a comment to the code explaining it first tries to use fog and fallbacks to ovirt, from first look it looks as typo :-)